frontend/src/metabase/css/core/base.styled.ts

```diff
@@ -3,7 +3,6 @@ import { css } from "@emotion/react";
 
 export const getRootStyle = (theme: Theme) => css`
   font-family: var(--mb-default-font-family), sans-serif;
-  font-size: 0.875em;
```

Comment:
I don't think this change is necessary as this `font-size` is defined well before the custom one, but I guess you removed it for clarity, so we don't define `font-size` twice in the SDK?

Reply:
Yes, this is just so we don't get a duplicate CSS declaration in the inspector to avoid confusion when debugging.